hnn_core/dipole.py

```diff
@@ -93,6 +93,7 @@ def simulate_dipole(net, n_trials=None, record_vsoma=False,
     # XXX needed in mpi_child.py:run()#L103; include fix in #211 or later PR
     net.params['N_trials'] = n_trials
     net._instantiate_drives(n_trials=n_trials)
```

Comment:
Are these needed here?

Duplicated here:
https://github.com/jonescompneurolab/hnn-core/blob/acc4a596db5fb6fc995d8902c6019cd23990a8dc/hnn_core/drives.py#L140

Reply:
Well... The `add_drives....`-argument was thought of as an intermediate step to deprecating it. Right now our tests are dependent on a `net` having all drive events defined. In actual use, we placed drive instantiation into `simulate_dipole`, because the `NetStim` times aren't really needed before then. I think we need to keep it in until we figure out our test suite going forward. But the one in `drives` is only called once when a `Network` object is created. Sure, it gets called again in `simulate_dipole`, but I think the overhead is minimal.

Reply:
Ok. Yeah, overhead is minimal.